src/app/compiler/compiler-sourceVerifier-fetchAndCompile.js

```diff
@@ -72,7 +72,7 @@ export default class FetchAndCompile extends Plugin {
     name === 'main' ? 'mainnet' : name // source-verifier api expect "mainnet" and not "main"
     let data
     try {
-      data = await this.call('source-verification', 'fetchByNetwork', contractAddress, name.toLowerCase())
+      data = await this.call('sourcify', 'fetchByNetwork', contractAddress, name.toLowerCase())
```

Comment:
if you di this in this PR please also change it on homepage UI and loading page code